### PR TITLE
[WIP] NServiceBus.RavenDB main package on netstandard

### PR DESCRIPTION
**Pull request by DavidBoike**
Note that RavenDB is not signing their netstandard1.3 libraries, so I had to mirror here.

### CODE REVIEW

File: src/NServiceBus.RavenDB/Internal/DocumentStoreManager.cs [hunk 1]
```diff
@@ -143,16 +152,35 @@ private static DocumentStoreInitializer CreateDefaultDocumentStore(ReadOnlySetti
...
         }
 
+        static DocumentStoreInitializer CreateStoreByUrl(ReadOnlySettings settings, string url)
+        {
+            var docStore = new DocumentStore
+            {
+                Url = url
+            };
+
+            if (docStore.DefaultDatabase == null)
```
Comment (SimonCropp) [inline src/NServiceBus.RavenDB/Internal/DocumentStoreManager.cs]: wouldnt this always be null?

File: src/NServiceBus.RavenDB/NServiceBus.RavenDB.csproj [hunk 2]
```diff
@@ -1,14 +1,17 @@
...
-    <TargetFramework>net452</TargetFramework>
+    <TargetFrameworks>net452;netstandard2.0</TargetFrameworks>
     <RootNamespace>NServiceBus.Persistence.RavenDB</RootNamespace>
-    <SignAssembly>true</SignAssembly>
     <AssemblyOriginatorKeyFile>..\NServiceBus.snk</AssemblyOriginatorKeyFile>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
   </PropertyGroup>
+  
+  <PropertyGroup Condition="'$(TargetFramework)' == 'net452'">
+    <SignAssembly>true</SignAssembly>
```
Comment (SimonCropp) [inline src/NServiceBus.RavenDB/NServiceBus.RavenDB.csproj]: why not sn the netstandard dll?

**Comment by SimonCropp** [comment]
in the next release (currently in prerelease) of the raven client the netstandard assembly is SN'd

**Comment by DavidBoike** [comment]
> in the next release (currently in prerelease) of the raven client the netstandard assembly is SN'd

Then I propose we backburner this until they release that patch as a non-prerelease version.

**Comment by bording** [comment]
Just noticing this now, but #383 would probably be the one to take now since it's more up to date.

**Comment by DavidBoike** [comment]
Closing in favor of #383

**Pull request closed without merging**